### ISSUE TITLE
Request: local instance database name

### ISSUE THREAD

**Issue by Gagankarwar** [microsoft/ReportingServicesTools#326]
Hi there, 
Thanks for great work done here. 

I wam working on a script and want to run external tool script outside power bi since that is not allowed in my official laptop. The functionality I am building is quiet straightforward but I couldn't figure how can I get database name of power bi local instance directly using PowerShell code. I can get local host name but can't go to get database name require to connect with datamodel.
Basically want %database% not coming from external tools in power bi but outside of that
Thanks in advance
